### DISCUSSION TITLE
part-ii/2018-74/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#91] [Question comments]
Part II 2018 General Relativity: Paper 3, Section II, E 

https://questions.tripos.org/part-ii/2018-74/

**Comment by AndoBando** [comment]
The transformation is a Lorentz boost. You know from special relativity that this means $-dt^2 + dx^2 + dy^2 + dz^2 = -d\bar{t}^2 + d\bar{x}^2 + d\bar{y}^2 + d\bar{z}^2$. 
If you write $ds^2 = (1+A)^4 (-dt^2 + dx^2 + dy^2 + dz^2) + \left((1+A)^4- \frac{(1-A)^2}{(1+A)^2} \right) dt^2$ and then do the substitution the algebra becomes much simpler.
